src/fidesops/ops/core/config.py

```diff
@@ -167,12 +174,14 @@ class FidesopsConfig(FidesSettings):
     database: FidesopsDatabaseSettings
     redis: RedisSettings
     security: FidesopsSecuritySettings
-    execution: ExecutionSettings
-    root_user: RootUserSettings
-    admin_ui: AdminUiSettings
-    notifications: FidesopsNotificationSettings
-
-    port: int
+    execution: Optional[ExecutionSettings] = ExecutionSettings()
```

Comment:
There's a (required!) config variable missing from this schema: fidesops.security.drp_jwt_secret. Can you add it to the schema and ensure it's optional?

Reply:
It's in the [Fideslib config](https://github.com/ethyca/fideslib/blob/main/fideslib/core/config.py#L97) already, so we shouldn't specify it twice

Reply:
Ah, I see! I'd like it to have a default though, can't we make that optional?

Reply:
Good spot, have made [this update to Fideslib](https://github.com/ethyca/fideslib/pull/75)